### ISSUE TITLE
Need clearer/designed debug output for pilots

### ISSUE THREAD

**Issue by george08** [museuminabox/brain#93]
The stuff that splats out of box.local:1880 debug tab is a little weird. Can we design that and put it in an iframe, with English/written responses that make a bit more sense to mere mortals?

E.g.
- OK! That sticker is written.
- Can't find the RFID tag. Please try again?
etc

**Comment by george08** [comment]
@thibaut-evrard is thinking about this too... wondering if we should pop box.local UI in an iframe in the heart system, too.

**Comment by amcewen** [comment]
Arguably we shouldn't be using the NodeRED debug output as a UI :-D

There'll be a load of it that we don't have any control over.  It feels like this is mostly a "we don't trust the output on `/prints`" rather than "we should expose the internal debug output to the end user", is that right?

The output on `/prints` when writing tags should be lots better since the redesign work in the run up to the big commission, but there might still be glitches/bugs in it.  Plus we could look at designing that better too, as is already mentioned in #73?

**Comment by george08** [comment]
Yep - agreed that the raw NodeRED is a little hard to swallow :)

I'm gathering bits I think we should design properly with a human in mind under this label: https://github.com/museuminabox/brain/labels/user%20interface

**Comment by amcewen** [comment]
it would be good to add audio versions of the success/error messages too.

**Comment by george08** [comment]
@amcewen what do you reckon is the right approach here, for the pilot? 

it is a little bit that "we don't trust the output on /prints", since the test of booping the newly minted object's sticker and expecting audio doesn't always work, and it isn't clear why. 

Is there a list of the debug messages anywhere? Is it the case that there is only one true "Success!" state of the process? If so, can /prints just wait for that one?

I guess this will be helped a bit by the "download all the audio you need" button on the /prints page, too, if some of the inconsistency is caused by a lack of audio.

**Comment by amcewen** [comment]
I'm pretty sure (would have to double-check the code, as it's a while since I wrote that) that `/prints` is only waiting for the one true "success" message at present.

I'm also fairly sure that the audio download failures are what's causing us not to trust the output on `/prints`.  I might be wrong, but from what you've described of it, that's the most logical option.

**Comment by amcewen** [comment]
With the better audio download stuff from #100 and #76 I think this will work better.  Not going to do any more on it for the pilot, but will leave it open so next time we're triaging we can confirm that we aren't still seeing problems with the tag writing flow.
